src/lib/TimeSeries/types.ts

```diff
@@ -24,3 +27,13 @@ export enum SeriesOperationType {
   Amplitude = 'amplitude',
   VerticalClearance = 'vertical-clearance',
 }
+
+export interface TimeSeriesDisplaySubplotItem extends SubPlotItem{
+  sequence?: number;
+}
+
+
+export interface TimeSeriesDisplaySubplot {
+  xAxis?: TimeSeriesDisplaySubplotItemAxis;
+  items: TimeSeriesDisplaySubplotItem[];
+}

```

Comment:
Should both be updated in the `fews-pi-request` library.

Reply:
is requested in https://issuetracker.deltares.nl/browse/FEWS-29590 and https://github.com/Deltares/fews-wms-requests/issues/21